net-neutrality/README.md

```diff
@@ -2,8 +2,34 @@
 
+Net neutrality is “the principle that Internet service providers and governments regulating the Internet should treat all data on the Internet the same, not discriminating or charging differentially by user, content, website, platform, application, type of attached equipment, or mode of communication.” (Wikipedia)
+
+This means that the companies and organisations which provide the underlying technical infrastructure of the internet must treat all data equally - not favouring or discriminating by content, source or destination.
+
 ## Why is it important?
 
+This concept is fundamental to ensuring that there is no discrimination, either positive or negative, for different types of data travelling over the Internet.
+
+Without net neutrality, Internet Servide Providers (ISPs) – and the governments which regulate them – would be free to engage in damaging practices such as:
+
+* Slowing down access to websites and online services which cannot – or will not – pay a premium to ISPs, creating a tax on consumer access;
+* Throttling or entirely blocking web sites and online services with which the ISP disagrees, whether that is a competitor’s services, charities, or health information;
+* Throttling or entirely blocking web sites as a result of government pressure, for example, social media sites and blogging platforms which host user content critical of that government.
+
 ## What's happening with it?
 
-## The Web Matters position
\ No newline at end of file
+Net neutrality is generally established, though not guaranteed, in Europe through Article 3 of [Regulation EU 2015/2120](http://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32015R2120&rid=2).
+
+However, in the United States – where many of the sites and services millions of web users rely on are based – net neutrality is under attack.
+
+Under the mantras of deregulation and market control, the Republican party has had the net neutrality provisions in their sights for some time, and began implementing plans for its removal as soon as Donald Trump was elected to office. There are two reasons for this. One is the general right-leaning adherence to libertarian market principles. The other is deep personal antipathy towards President Obama, under whose administration the net neutrality principles were enshrined.
+
+In the US, the regulation of telecommunications, including internet service providers, falls to the Federal Communications Commission (FCC). Under the Trump presidency, the FCC’s moves to dismantle net neutrality have escalated fast.
+
+This is not a US-specific thing. In the UK certain mobile phone providers have been offering deals where data from certain websites - Netflix, Soundcloud, Deezer and others - will not come out of the user’s data allowance. This is not because of the generosity of the mobile phone providers, but because of deals done with these websites. This tipping of the playing field will have huge negative effects on smaller media providers.
```

Comment:
Done, but you you have a link to the source which says OFCOM is investigating it?

Related: looking through the recent news from OFCOM shows what a cesspool the communications industry is. #shudder